src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -37,6 +37,10 @@ public string ZipUrl
                 {
                     return Environment[EnvironmentSettingNames.AzureWebsiteZipDeployment];
                 }
+                else if (Environment.ContainsKey(EnvironmentSettingNames.ScmBuildPackage))
+                {
+                    return Environment[EnvironmentSettingNames.ScmBuildPackage];
```

Comment:
Is the format of this a URL to a zip package as well?

Reply:
Yes, this will be populated with the URL of the built zip.